examples/hypothesis_testing/plot_MI_imbalanced_hyppo_testing.py

```diff
@@ -91,18 +91,19 @@ def make_multiview_classification(
 
 
 n_samples = 100
-n_features = 10000
 n_features_1 = 10
-n_features_2 = n_features
+n_features_2 = 10_000
```

Comment:
Why does this have a underscore?

Reply:
To make it easier to read that it is `10_000` and remove the unnecessary variable `n_features`